mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
```

Comment:
The response message should be consistent throughout the API. In other parts of the code, error responses are sent with a JSON object containing an "error" key. This response should follow the same pattern. It should be changed to `res.json({ message: "Order deleted successfully", success: true });` or `res.json({ success: true, message: "Order deleted successfully" });` to maintain consistency.